### PR TITLE
[WIP] SES-45 Connect CU table to the API

### PR DESCRIPTION
**Pull request by z3phyro**
# Ticket
[SES-45](https://trello.com/c/Vu4HjfsA/30-connect-cu-table-to-the-api)

# Actions
Removed Apollo Client
Added graphql-request
Created necessary structure to parse the data from the API
Created methods to parse/process the data from the server
Improved state management for the cu-table
Created scenario for expenditures column

### CODE REVIEW

File: src/stories/components/svg/twitter.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ const Twitter = ({
 }: Props) => {
   return (
     <svg width={width} height={height} viewBox={`0 0 ${width} ${height}`} fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (alejandrocabriales) [inline src/stories/components/svg/twitter.tsx:14]: Is there a reason for change the viewBox?

File: src/stories/containers/cutable/cutable.tsx [hunk 2]
```diff
@@ -9,76 +9,71 @@ import { CutableColumnSummary } from '../../components/cutable-column-summary/cu
...
-          {
-            linkType: LinkType.LinkedIn,
-            href: '#',
-          },
-        ]}
-      />
-    ]);
-  };
+  useEffect(() => {
+    dispatch(loadAsync());
```
Comment (alejandrocabriales) [inline src/stories/containers/cutable/cutable.tsx:44]: Could you provide a more descriptive name for this function

File: src/stories/containers/cutable/cutable.tsx [hunk 3]
```diff
@@ -9,76 +9,71 @@ import { CutableColumnSummary } from '../../components/cutable-column-summary/cu
...
+  }, []);
+
+  const items = useMemo(() => {
+    if (!data) return [];
+    console.log(data);
```
Comment (alejandrocabriales) [inline src/stories/containers/cutable/cutable.tsx]: Could you please remove the console.log(data);

Comment (z3phyro) [reply src/stories/containers/cutable/cutable.tsx]: the console.log was removed a couple of commits after that 
image

In Added client side filter and search

File: src/stories/components/svg/twitter.tsx (hunk 1 above)
Comment (z3phyro) [reply src/stories/components/svg/twitter.tsx:14]: I thought you were fixing this in your commit weren't you? It's not part of the changes I'm doing, If I make that change we will have a conflict when merging.

File: src/stories/containers/cutable/cutable.tsx (hunk 2 above)
Comment (z3phyro) [reply src/stories/containers/cutable/cutable.tsx:44]: Sure thing. Done.

File: src/stories/containers/cutable/cutable.tsx (hunk 3 above)
Comment (alejandrocabriales) [reply src/stories/containers/cutable/cutable.tsx]: perfect

File: src/stories/containers/cutable/cutable.tsx (hunk 2 above)
Comment (alejandrocabriales) [reply src/stories/containers/cutable/cutable.tsx]: perfect

File: src/stories/components/svg/twitter.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/stories/components/svg/twitter.tsx]: ok, i will fix my pr, to let without  viewBox

**Pull request merged**